cpp/src/IceStorm/Instance.cpp

```diff
@@ -289,25 +281,23 @@ Instance::destroy()
 
 PersistentInstance::PersistentInstance(
     const string& instanceName,
-    const string& name,
     shared_ptr<Ice::Communicator> communicator,
     Ice::ObjectAdapterPtr publishAdapter,
     Ice::ObjectAdapterPtr topicAdapter,
     Ice::ObjectAdapterPtr nodeAdapter,
     optional<NodePrx> nodeProxy)
     : Instance(
           instanceName,
-          name,
           communicator,
           std::move(publishAdapter),
           std::move(topicAdapter),
           std::move(nodeAdapter),
           std::move(nodeProxy)),
-      _dbLock(communicator->getProperties()->getPropertyWithDefault(name + ".LMDB.Path", name) + "/icedb.lock"),
+      _dbLock(communicator->getProperties()->getPropertyWithDefault("IceStorm.LMDB.Path", "IceStorm") + "/icedb.lock"),
```

Comment:
I think we should keep the service name here. And add a helper function to get the LMDB path, something like:

```cpp
string getLMDBPath(const PropertiesPtr& properties, const string& serviceName)
{
    string path = properties->getIceProperty("IceStorm.LMDB.Path");
    if (path.empty())
    {
        path = serviceName;
    }
    return path;
}
```